src/components/LoginForm/LoginForm.tsx

```diff
@@ -0,0 +1,54 @@
+import { useRef, FormEvent } from 'react';
+import { Link } from 'react-router-dom';
+
+import { FormInput } from '../FormInput/FormInput';
+import style from './LoginForm.module.scss';
+
+export const LoginForm = () => {
+  const formRef = useRef<HTMLFormElement>(null);
+
+  const handleSubmit = (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+
+    if (!formRef.current) {
+      return;
+    }
+
+    const formData = new FormData(formRef.current);
```

Comment:
Lol. I never thought of using this. I did that like an absolute baboon, getting each of the values by name from the refs. 🤯 

TIL!